Packs/AzureSentinel/TestPlaybooks/NonCircleTests/playbook-TestAzureSentinelPlaybook.yml

```diff
@@ -13,6 +13,7 @@ tasks:
       name: ""
       iscommand: false
       brand: ""
+      description: ''
```

Comment:
Please revert the changes in this playbook

Reply:
done - want to mention that changes were made by demisto sdk, not a manual edit